src/main/scala/leon/frontends/scalac/ASTExtractors.scala

```diff
@@ -223,7 +244,7 @@ trait ASTExtractors {
     }
 
 
-
+    /** Returns a string literal either from leon.lang.string.String or from a constant string literal. */
     object ExStringLiteral {
       def unapply(tree: Tree): Option[String] = tree  match {
         case Apply(ExSelected("leon", "lang", "string", "package", "strToStr"), (str: Literal) :: Nil) =>
```

Comment:
This is no longer possible right? since you removed this conversion from the library.